CONTRIBUTING.md

```diff
@@ -1,7 +1,142 @@
 # Contributing Guidelines
 The OSM project accepts contributions via GitHub pull requests. This document outlines the requirements for contributing to this project.
 
-## Sign Your Work
+## Pull request workflow
+
+The following sections describe how to contribute code by opening a pull request.
+
+### 1. Fork the [Open Service Mesh](https://github.com/openservicemesh/osm) repository
+
+1. Visit `https://github.com/openservicemesh/osm`.
+1. Click the `Fork` button.
+
+### 2. Clone your fork
+Set `username` to match your Github username:
+```
+export username=<github username>
+```
+
```

Comment:
I am not certain that we need the GitHub username exported to the environment.
If we do - I'd recommend using `GITHUB_USERNAME` so that this is a bit more specific what username that is.

```suggestion
```

Reply:
The username is referenced in subsequent sections in this doc, so I'll change this to `GITHUB_USERNAME` as you've suggested.